tests/unit/s2n_client_hello_test.c

```diff
@@ -791,68 +790,86 @@ int main(int argc, char **argv)
 
         uint8_t client_extensions[] = {
             /* Extension type TLS_EXTENSION_SERVER_NAME */
-            0x00, 0x00,
+            0x00,
```

Comment:
How come we can keep the line breaks sometimes (like on line 100 in this file), but it gets put into a single column here?

Reply:
Its to do with the placement of the comments. On line 100 the comments come at the end of a multi-element line.

So we could reformat this to:
```c
         uint8_t server_name_extension[] = {
            0x00, 0x06,  /* Server names len */
            0x00,             /* First server name type - host name */
            0x00, 0x03,  /* First server name len */
            's', 'v', 'r',      /* First server name, matches sent_server_name */
        };
```

And clang-format would accept it.

Reply:
We determined this change would require to many manual changes.